pegasus/sites.v3/hourofcode.com/public/index.haml

```diff
@@ -9,6 +9,11 @@ social:
 
 %link{href: "/css/highlights-homepage.css", rel: "stylesheet", id: "highlight-styles"}
 
+-# These are needed for the signup form, even if the map isn't currently showing.
+%script{src:'https://api.mapbox.com/mapbox-gl-js/v1.0.0/mapbox-gl.js'}
+:javascript
+  mapboxgl.accessToken = "#{CDO.mapbox_access_token}";
```

Comment:
the boundary of ruby in HAML files is somewhat confusing to me, but should this line use JS string interpolation instead of ruby? it could also be simplified to

```haml
:javascript
  mapboxgl.accessToken = CDO.mapbox_access_token;
```

unless we're also using this line to ensure the token value is stringified

Reply:
`CDO.mapbox_access_token` comes from Ruby, so we need `#{}` to pull it into what's otherwise some JavaScript.  And we wrap it in `""`s so that it gets parsed & stored as a string in the JavaScript.